gtk/src/ghb3.ui

```diff
@@ -8989,7 +8994,7 @@ Uncheck this if you want to allow changing each title's settings independently.<
                           <object class="GtkImage" id="live_preview_play_image">
                             <property name="visible">True</property>
                             <property name="can_focus">False</property>
-                            <property name="icon_name">gtk-media-play</property>
+                            <property name="icon_name">media-playbck-start</property>
```

Comment:
I don't know why GitHub is being screwy on me — I can't view this file diff in the full set of PR changes, and I can't leave a suggested change with this comment — but:

Typo. this should be:

```xml
                            <property name="icon_name">media-playback-start</property>
```
(Or, after the changes in the next commit...)
```xml
                            <property name="icon-name">media-playback-start</property>
```